aldryn_forms/models.py

```diff
@@ -167,7 +170,14 @@ def get_form_elements(self):
         from .utils import get_nested_plugins
 
         if self.child_plugin_instances is None:
-            self.child_plugin_instances = self.get_descendants().order_by('tree_id', 'level', 'position')
+            # 3.1 and 3.0 compatibility
+            if USES_TREEBEARD:
+                # default ordering is by path
```

Comment:
to make it simpler, I'd make the condition set ordering as variable and then below just one call to get_descendants()